src/main/scala/uk/gov/hmrc/hmrcfrontend/views/Aliases.scala

```diff
@@ -101,6 +101,9 @@ trait Aliases {
 
   type AddToAList = viewmodels.addtoalist.AddToAList
   val AddToAList = viewmodels.addtoalist.AddToAList
+
+  type Timeline = viewmodels.timeline.Timeline
```

Comment:
Do we need an alias for Event too?

Reply:
Updated here https://github.com/hmrc/play-frontend-hmrc/pull/77/commits/ea82f4b7c81cccaede4d1d55f2701f17e7893337